site/en/docs/extensions/mv3/override/index.md

```diff
@@ -64,17 +75,24 @@ For `PAGE_TO_OVERRIDE`, substitute one of the following:
 
 For an effective override page, follow these guidelines:
 
-- **Make your page quick and small.**
-  Users expect built-in browser pages to open instantly. Avoid doing things that might take a long
-  time. For example, avoid synchronous fetches of network or database resources.
-- **Include a title in your page.**
-  Otherwise people might see the URL of the page, which could be confusing. Here's an example of
-  specifying the title: `<title>New Tab</title>`
-- **Don't rely on the page having the keyboard focus.**
-  The address bar always gets the focus first when the user creates a new tab.
-- **Don't try to emulate the default New Tab page.**
-  The APIs necessary to create a slightly modified version of the default New Tab page—with top
-  pages, recently closed pages, tips, a theme background image, and so on—don't exist.
+**Make your page quick and small.**
+
+Users expect built-in browser pages to open instantly. Avoid doing things that might take a long
+time. For example, avoid synchronous fetches of network or database resources.
+
+**Include a title in your page.**
+
+Otherwise people might see the URL of the page, which could be confusing. Here's an example of
```

Comment:
```suggestion
**Include a title in your page.**<br/>Otherwise people might see the URL of the page, which could be confusing. Here's an example of
```